web/src/components/Plugin/locales/zh-CN.ts

```diff
@@ -21,4 +21,7 @@ export default {
   'component.step.select.pluginTemplate.select.option': '手动配置',
   'component.plugin.pluginTemplate.tip1': '1. 若路由已配置插件，则插件模板数据将与已配置的插件数据合并。',
   'component.plugin.pluginTemplate.tip2': '2. 插件模板相同的插件会覆盖掉原有的插件。',
+
+  // proxxy-mirror
+  'component.pluginForm.proxy-mirror.host.tooltip': '指定镜像服务地址，例如：http://127.0.0.1:9797（地址中需要包含 schema ：http或https，不能包含 URI 部分）',
```

Comment:
After some searches, I noticed there should be `Path` instead of `URI`?

In APISIX, we use `URI` to refer to `HTTP Request Path`, it's not appropriate IMO.

```suggestion
  'component.pluginForm.proxy-mirror.host.tooltip': '指定镜像服务地址（如 http://127.0.0.1:9797），地址中需包含协议：http 或 https，不能包含 PATH 部分）',
```

Reply:
IMO, we should keep the documentation consistent, otherwise, it can confuse users when explanations are inconsistent. I've this in this issue https://github.com/apache/apisix-dashboard/issues/1724.